lib/diaspora/mentionable.rb

```diff
@@ -75,7 +75,7 @@ def self.backport_mention_syntax(text)
     text.to_s.gsub(NEW_SYNTAX_REGEX) do |match_str|
       _, diaspora_id = mention_attrs(match_str)
       person = find_or_fetch_person_by_identifier(diaspora_id)
-      old_syntax = "@{#{person.name}; #{diaspora_id}}" if person
+      old_syntax = "@{#{person.name.tr('{}', '')}; #{diaspora_id}}" if person
```

Comment:
Maybe someone with more Ruby skills should have a look at this but `person.name.delete('{}')` looks more obvious to me.